src/java/grl/src/grl/driver/GRL_Driver.java

```diff
@@ -501,26 +509,71 @@ else if (_currentKUKAiiwaState.armControlState().stateType() == grl.flatbuffer.A
 				System.out.println("Unsupported Mode! stopping");
 				stop = true;
 			}
-            
+
+
+			/// Reading sensor values from Java Interface and sending them thrugh ZMQ
+			if (_currentKUKAiiwaState.armControlState().stateType() == grl.flatbuffer.ArmState.MoveArmJointServo){
+
+
+			Vector force = _lbr.getExternalForceTorque(_lbr.getFlange()).getForce();
+
+			double force_x = force.getX();
```

Comment:
Yes, We should add other stuff too, and add flags for them... also we should try reading them in other threads as well, see if it helps the performance